es6/README.md

```diff
@@ -704,6 +704,69 @@ Button.propTypes = {
 
 Both declarative property syntaxes are not a part of the ES2015 specification and are a in the midst of the ECMAScript proposal approval process. Currently they are sitting in Stage 3. For details, check out [ECMAScript Class Fields and Static Properties](https://github.com/jeffmo/es-class-fields-and-static-properties).
 
+For bound methods within classes, we recommend to use fat arrow syntax to write functions need current context:
+
+```js
+// good
+class Button extends React.Component {
+    _handleOnClick = () => {
+        //handle button click.
+    }
+
+    render() {
```

Comment:
Should `render` (and other lifecycle methods) also be bound too?

```
render = () => (
    <button onClick={this._handleOnClick} />
)
```

Reply:
I like only using this functions if the function needs to be bound, e.g. event handlers, since this syntax is more concise and reduce the risk that some issues raised by `.bind`.